Floofbot/Program.cs

```diff
@@ -21,17 +20,7 @@ static async Task Main(string[] args)
         {
             InitialiseLogger();
 
-            // TODO: Replace console input with env variable import
-            string token;
-            if (!(args.Length == 1))
-            {
-                Console.WriteLine("Enter Bot Token");
-                token = Console.ReadLine();
-            }
-            else
-            {
-                token = args[0];
-            }
+            string token = ConfigurationManager.AppSettings["Token"];
```

Comment:
maybe add a check here for a blank token and tell the user how to modify it?

Reply:
good idea -- done